data/records/delphi-conditions-database.json

```diff
@@ -0,0 +1,75 @@
+[
+  {
+    "abstract": {
+      "description": "The DELPHI conditions database."
+    },
+    "accelerator": "CERN-LEP",
+    "collaboration": {
+      "name": "DELPHI collaboration",
+      "recid": "80050"
+    },
+    "collections": [
+      "DELPHI"
```

Comment:
The value should rather be `DELPHI-Condition-Data` to have parity with CMS.

(Note also singular instead of plural.)